docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-penaltymodel-maxgap==0.5.4
+penaltymodel-maxgap==0.5.5
 
 # June 15, 2021: temporary docutil pinning for https://github.com/readthedocs/sphinx_rtd_theme/issues/1115
 docutils==0.16 
```

Comment:
Looks like https://github.com/readthedocs/sphinx_rtd_theme/issues/1115 has been closed. Up to you if you want to try for this release or leave for later testing and then incorporation.